rustworkx/__init__.pyi

```diff
@@ -15,213 +15,6 @@ import rustworkx.visit as visit
 from .rustworkx import *
 from typing import Generic, TypeVar, Any, Callable, Iterator, overload
 
-from .graph import PyGraph as PyGraph
```

Comment:
You are going to need to re-export those with `from .rustworkx import PyGraph as PyGraph` because of  #960

Reply:
Done in: https://github.com/Qiskit/rustworkx/pull/1061/commits/85672db3dd3a87726856983c6996896e69c0f1d6